src/task_list/moduls.py

```diff
@@ -0,0 +1,146 @@
+import json
+import pathlib
+
+
+class DataJson:
+    'Класс для работы с файлом'
+
+    def __init__(self):
+        self.file = pathlib.Path('task_list/src/config/data.json').resolve()
+
+
+    def read_data_json(self):    # Чтение файла с задачами
+        with open(self.file, 'r', encoding='utf-8') as file:
+            tasks = json.load(file)
+
+        return tasks
+
+
+    def write_to_file(self, task):    # Запись задач в файл
+        with open(self.file, 'w', encoding='utf-8') as file:
+            json.dump(task, file, indent=4)
+
+
+    def get_task_id(self):   # Получение id последней задачи
+        id_task = list(self.read_data_json().keys())
+
+        if len(id_task) == 0:
+            return 0
+
+        return int(id_task[-1])
+
+
+
+class Tasks(DataJson):
+    'Класс для работы с задачами'
+
+    def __init__(self):
+        self.tasks = {}
+        self.task_id = 1
+        super().__init__()
+
+
+    def get_all_tasks(self):    # Просмотр всех задач
+        if not self.read_data_json():
+            print('Список задач пуст.')
+        else:
+            for task_id, task in self.read_data_json().items():
+                print(f'Задача #{task_id}: {task}')
```

Comment:
nit: тут как хочешь, просто удобно + часто используется, когда ты проверил, что задач нет и сделал return - вышел из функции, а если не вышел, то код работает дальше

```suggestion
    def get_all_tasks(self):    # Просмотр всех задач
        if not self.read_data_json():
            print('Список задач пуст.')
            return
        for task_id, task in self.read_data_json().items():
            print(f'Задача #{task_id}: {task}')
```